src/util/silence-mode.js

```diff
@@ -10,6 +10,13 @@
  * @since 5.2.0
  */
 export default function ignoreSilenceMode() {
+    // Set webaudio context with 1 second silent audio 44100 bit rate buffer to allow playing audio even if silent switch is on the device
+    let silentAC = new AudioContext();
+    let silentBS = silentAC.createBufferSource();
```

Comment:
The `let` can be changed to `const` since you're not modifying these variables further on.

Reply:
@katspaugh updated, yes the fix for mediaelement worked for webaudio as well to allow audio to play based on user interaction. However on latest updates of ios if the silent switch is on it will detect it's playing a mediaelement source so it blocks webaudio as it's not technically triggered by user interaction. To fix this a webaudio buffer needs to be played prior as well as mediaelement so it detects webaudio is played allowing the audio to continue.

Reply:
Excellent, thank you!